<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WashModePage</class>
 <widget class="QWidget" name="WashModePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget

{
	background-color: rgb(51,153,255);
}

QLabel
{
border:1px solid black;
font:15px , &quot;SimHei&quot;;
color:black;
}
QPushButton
{
outline:none;
font:25px, &quot;SimHei&quot;;
border-image: url(:/image/buttonUP.jpg);
color:rgb(0, 32, 96);
}

QPushButton:disabled
{
outline:none;
border:1px solid white;
border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(180, 180, 180);
border-radius:2px;
}

</string>
  </property>
  <widget class="QPushButton" name="btnWash">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>洗车模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>390</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoCondition">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
color:red;
font:17px;</string>
   </property>
   <property name="text">
    <string>洗车条件不具备</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
color:red;
font:17px;</string>
   </property>
   <property name="text">
    <string>洗车进行中</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
